src/core/SampleClip.cpp

```diff
@@ -260,7 +260,7 @@ void SampleClip::saveSettings( QDomDocument & _doc, QDomElement & _this )
 	_this.setAttribute( "len", length() );
 	_this.setAttribute( "muted", isMuted() );
 	_this.setAttribute( "src", sampleFile() );
-	_this.setAttribute( "off", startTimeOffset() );
+	_this.setAttribute("offset", startTimeOffset());
```

Comment:
This attribute name needs to stay the same for backwards compatibility. (Alternatively, you could write a new upgrade routine to change it, but I'd rather we did that separately and fixed a bunch of other names while we were at it.) I'm also inclined to use "off" for now in `PatternClip` for consistency, but I don't mind so much about that.

Reply:
undone